pyproject.toml

```diff
@@ -53,6 +53,10 @@ myst-parser = {version = ">=0.16.1"}
 [tool.poetry.scripts]
 wordcab = "wordcab.__main__:main"
 
+[tool.poetry.group.dev.dependencies]
+nox = "^2022.11.21"
+nox-poetry = "^1.0.2"
+
```

Comment:
We won't include `nox` and `nox-poetry` in the dev dependencies as they should be installed system-wide by the developer collaborating on the project.

You pointed out something interesting, the way we declared `dev-dependencies` has been deprecated since poetry 1.2.0 (as stated in [poetry doc](https://python-poetry.org/docs/master/managing-dependencies/#dependency-groups))

We should update the dev tag to avoid extra warnings.